README.md

```diff
@@ -2,36 +2,61 @@
 
-The game master will have features to easely handle the means of a P&P RPG. We also want to provide a global hub in which you can join a session whenever the game master is able to introduce you to the game.
+The game master will have features to easely handle the means of a P&P RPG (features yet to be defined). We also want to provide a global hub in which you can join a session whenever the game master is able to introduce you to the game.
 
-For the first version, we do not want to force users to create accounts.
+For the first version, we do not want to force users to create accounts, kind of like IRC. TODO: but will it be available?
 
 ## Contributing
 
 Feel free to create issues and/or pull requests, we would love your contribution ! There's no small help :)
-Remember following Elixir's syntax, and good practices.
+We still didn't write any CONTRIBUTING guide, but writing some clean elixir code, formated with the default formatter (`mix format`), is a rather good start.
 
 ## Setup
 
-To create your PostgreSQL Database we recommend that you use [Postgres app](https://postgresapp.com/) on mac.
+### Requirements 
 
-To start your Phoenix server:
+You'll need to install [`erlang`], [`elixir`], the [`phoenix framework`][`phoenix`], and [`postgresql`].
 
-  * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
-  * Start Phoenix endpoint with `mix phx.server`
+For [`elixir`] and [`erlang`] follow the instructions on the [elixir install page](https://elixir-lang.org/install.html). We use a minimal version of v1.7 for [`elixir`], which in turn requires a minimal version of v19.0 for [`erlang`]. You can check for the versions by running the following command:
+```bash
+$ elixir --version
+Erlang/OTP 19 [erts-8.0] [64-bit] [smp:4:4] [async-threads:10]
+
+Elixir 1.7.1 (compiled with Erlang/OTP 19)
+```
+For [`phoenix`], run the following commands in your shell (or powershell for windows users):
+```bash
+# only once, will install the package manager of elixir
+# might ask for confirmation, if it does, do not say yes, it means you've already did it
+$ mix local.hex
+
+# install phoenix
+$ mix archive.install https://github.com/phoenixframework/archives/raw/master/phx_new.ez
+```
+If it fails, please have a look at <https://hexdocs.pm/phoenix/installation.html>
+
+For [`postgresql`], install using your prefered way, and setup the DB to match the config in [`config/dev.exs`](./config/dev.exs).
+
+> :warning: This parts needs some serious changes, because right now, the config for postgresql is right in the repository, checked in, absolutly not flexible. We're still looking for a standard solution, matching what's being done in the real world (`env` variables, for exemple), but which still looks fine in regard to the best practices in the `erlang/OTP` world.
```

Comment:
The [RealWorld app for elixir](https://github.com/gothinkster/elixir-phoenix-realworld-example-app) use environment variables for prod environment (eg `System.get_env("DATABASE_URL")`, and provides no `config/dev.exs`, but a template, that you must copy and then edit to your need.
The standard for nearly any other kind of app is [`dotenv`](https://www.google.com/search?hl=en&q=dotenv), but it looks like it [is'nt the elixir way](https://github.com/avdi/dotenv_elixir), and so should be avoided.
So I think we should be using something like the `config/prod.secret.exs`, but for `dev`, which won't be checked in.
Anyway, that's a problem that needs to be solved.

Reply:
I understand what you're saying here, feel free to create another pull request to make that change, as i see no problem with it